File: updateDb.js
import pkg from 'pg';
const { Client } = pkg;

const client = new Client({
  connectionString: "postgres://neondb_owner:npg_0ItvywJCB4RX@ep-tiny-mode-aqaqluvi-pooler.c-8.us-east-1.aws.neon.tech/neondb?sslmode=require",
});

async function run() {
  await client.connect();
  console.log("Connected to Neon DB");

  try {
    await client.query('ALTER TABLE "Artworks" ADD COLUMN "OriginalCoverUrl" text NULL;');
    console.log("Added OriginalCoverUrl to Artworks");
  } catch (e) {
    console.log("Artworks alter error:", e.message);
  }

  try {
    await client.query('ALTER TABLE "Messages" ADD COLUMN "Status" text NULL;');
    console.log("Added Status to Messages");
  } catch (e) {
    console.log("Messages alter error:", e.message);
  }

  // Also check if Prisma created it with lowercase 'artworks'
  try {
    await client.query('ALTER TABLE artworks ADD COLUMN "original_cover_url" text NULL;');
    console.log("Added original_cover_url to artworks (lowercase)");
  } catch (e) {
    console.log("artworks (lowercase) alter error:", e.message);
  }

  try {
    await client.query('ALTER TABLE messages ADD COLUMN status text NULL;');
    console.log("Added status to messages (lowercase)");
  } catch (e) {
    console.log("messages (lowercase) alter error:", e.message);
  }

  await client.end();
}

run();

File: archive_scripts/UpdateDb.cs
using System;
using Npgsql;

class Program
{
    static void Main()
    {
        var connString = "Host=ep-tiny-mode-aqaqluvi-pooler.c-8.us-east-1.aws.neon.tech;Database=neondb;Username=neondb_owner;Password=npg_0ItvywJCB4RX;Ssl Mode=Require;Trust Server Certificate=true;";
        using var conn = new NpgsqlConnection(connString);
        conn.Open();

        try {
            using var cmd1 = new NpgsqlCommand("ALTER TABLE \"artworks\" ADD COLUMN \"original_cover_url\" text NULL;", conn);
            cmd1.ExecuteNonQuery();
            Console.WriteLine("Added original_cover_url to artworks");
        } catch (Exception e) {
            Console.WriteLine("Artworks error: " + e.Message);
        }

        try {
            using var cmd2 = new NpgsqlCommand("ALTER TABLE \"messages\" ADD COLUMN \"status\" text NULL;", conn);
            cmd2.ExecuteNonQuery();
            Console.WriteLine("Added status to messages");
        } catch (Exception e) {
            Console.WriteLine("Messages error: " + e.Message);
        }
    }
}
